go.mod

```diff
@@ -10,13 +10,16 @@ require (
 	github.com/go-openapi/strfmt v0.21.2
 	github.com/go-openapi/swag v0.21.1
 	github.com/google/go-cmp v0.5.8
+	github.com/patrickmn/go-cache v2.1.0+incompatible
 	github.com/pborman/getopt v0.0.0-20180811024354-2b5b3bfb099b
 	github.com/sigstore/cosign v1.8.1-0.20220601172726-ae90c7495df6
 	github.com/sigstore/fulcio v0.1.2-0.20220114150912-86a2036f9bc7
 	github.com/sigstore/rekor v0.8.0
 	github.com/sigstore/sigstore v1.2.1-0.20220614141825-9c0e2e247545
 	golang.org/x/crypto v0.0.0-20220411220226-7b82a4e95df4
 	golang.org/x/oauth2 v0.0.0-20220524215830-622c5d57e401
+	google.golang.org/grpc v1.47.0
```

Comment:
Any big reason to prefer gRPC+proto over HTTP+JSON here? You're basically just POSTing some bytes and GETting some bytes, proto seems like a large hammer for that. Unless you have broader plans for this in the future?

Reply:
Moved to net/rpc for now - HTTP seemed overkill for something that would only ever be local (grpc at least codegens away much of the complexity of setting up the client/server).